### PR TITLE
Add option to skip a range from diff comparison

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Skip maven metadata from comparison when revisions are diffed

### CODE REVIEW

**Comment by praveenkuttappan** [comment]
I have deployed this change in staging instance. Java maven properties are visible now for reviews.

File: src/dotnet/APIView/APIView/CodeFileRenderer.cs [hunk 1]
```diff
@@ -56,6 +61,19 @@ private void Render(List<CodeLine> list, IEnumerable<CodeFileToken> node, bool s
                         isDeprecatedToken = false;
                         break;
 
+                    case CodeFileTokenKind.SkipDiffRangeStart:
+                        // Skip only when creating diff text
+                        // Tokens should not be skipped from html renderring
+                        if (GetType() == typeof(CodeFileRenderer))
```
Comment (pakrym) [inline src/dotnet/APIView/APIView/CodeFileRenderer.cs:67]: Let's at least encapsulate this logic in a virtual property.

**Comment by praveenkuttappan** [comment]
@JonathanGiles One question I have is: Should we skip entire maven section from comparison or only version info?

File: src/dotnet/APIView/APIView/CodeFileRenderer.cs (hunk 1 above)
Comment (praveenkuttappan) [reply src/dotnet/APIView/APIView/CodeFileRenderer.cs]: Done.

**Comment by praveenkuttappan** [comment]
@pakrym : Can you please have a look at this PR when you get a chance?

**Comment by JonathanGiles** [comment]
Can and should we apply this to the JavaDoc sections of our generator too, so that JavaDoc changes are not considered an API change?

**Review by pakrym — APPROVED** [review]

**Comment by praveenkuttappan** [comment]
> Can and should we apply this to the JavaDoc sections of our generator too, so that JavaDoc changes are not considered an API change?

I would like to handle this in separate PR since this change(If I include in same PR) will have impact on UI side as well. I assume you want to skip comparing doc part but UI diff option should still show difference in doc.

**Pull request merged**